.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[Feature Request] Title"
```

Comment:
Similar feedback as other template. Can we use labels instead of the bracket prefixes?

Reply:
I went ahead and added a label for feature requests as well.